course7/homeworks/patrickh/homework3/calculator.cpp

```
@@ -13,56 +13,56 @@ void Calculator::parse(const std::string& input)
     else if (input == "/")
         divide();
     else
-        mStack.emplace(input);
+        mStack.push(input);
 }
 
 void Calculator::add()
 {
     if(mStack.size() < 2)
-        throw std::runtime_error("Not enough values to be added");
+        throw new std::runtime_error("Not enough elements on the stack");
```

WRONG! you want to throw an exception, not a pointer to exception.